cfbs/commands.py

```diff
@@ -506,7 +506,7 @@ def validate_command():
     return 0
 
 
-def init_build_folder(config):
+def init_build_folder():
```

Comment:
Looks like a piece of change from the next commit, but most likely not worth moving to it.

Reply:
Thanks, I fixed it :)